src/index.html

```diff
@@ -40,16 +39,31 @@
       padding-top: 50px;
       padding-bottom: 50px;
     }
+    .col-md-4 {
```

Comment:
Instead of redefining a Bootstrap class, it would be better to make up a different class and apply these settings there. The element you're styling can have multiple classes.